x-pack/plugins/endpoint/scripts/alert_mapping.json

```diff
@@ -394,7 +394,8 @@
                 "type": "nested"
               },
               "file_extension": {
-                "type": "long"
+                "ignore_above": 1024,
```

Comment:
hmm 🤔 I thought these changes were already in master (same for event_mapping)?

https://github.com/elastic/kibana/blob/master/x-pack/plugins/endpoint/scripts/alert_mapping.json#L397

Maybe it's that your branch hasn't pulled in the latest changes.

Reply:
Yeah, I merged upstream and this went away. Thanks for the spot, Jonny.